erpnext/tests/test_italy_regional_patch.py

```
@@ -0,0 +1,267 @@
+			frappe.db.sql_ddl("ALTER TABLE `tabCustomer` ADD COLUMN `first_name` VARCHAR(140)")
+		if not frappe.db.has_column("Customer", "last_name"):
+			frappe.db.sql_ddl("ALTER TABLE `tabCustomer` ADD COLUMN `last_name` VARCHAR(140)")
+		frappe.clear_cache()  # Clear cache after adding columns
+
+	def _drop_old_columns_if_exist(self):
+		"""Drop old columns if they still exist."""
+		frappe.clear_cache()  # Clear cache to get fresh column info
+		try:
+			if frappe.db.has_column("Customer", "first_name"):
+				frappe.db.sql_ddl("ALTER TABLE `tabCustomer` DROP COLUMN `first_name`")
+		except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)